src/interface/variable.hpp

```diff
@@ -48,9 +48,11 @@ class CellVariable {
  public:
   /// Initialize a 6D variable
   CellVariable<T>(const std::string &label, const std::array<int, 6> dims,
-                  const Metadata &metadata)
+                  const Metadata &metadata, int sparse_id = -1)
       : data(label, dims[5], dims[4], dims[3], dims[2], dims[1], dims[0]),
-        mpiStatus(false), m_(metadata), label_(label) {
+        mpiStatus(false), m_(metadata),
+        label_(label + (sparse_id >= 0 ? "_" + std::to_string(sparse_id) : "")),
```

Comment:
Good point. We can't use NaN because it's an int. But I'll make a static constant `invalid_sparse_id` (probably `-INT_MAX` or something like that). Then we could also add support for negative sparse ids. Why not :)

Reply:
Actually, I think this change will better fit into #383, because I'm already changing and renaming some interfaces there.